config/packages/parameters.yaml.dist

```diff
@@ -0,0 +1,4 @@
+parameters:
+    app.authorization_token:
+    app.opaque_id:
+    app.readid_endpoint:
```

Comment:
Suggestion to rename this to readid hostname or something like that. As the endpoint itself is configured in the http clients.